node-red-node-ui-webcam/ui_webcam.js

```diff
@@ -221,6 +221,7 @@ module.exports = function(RED) {
                         $scope.changeCamera = function(deviceId) {
                             oldActiveCamera = activeCamera;
                             activeCamera = $scope.data.cameras[deviceId].deviceId;
+                            window.localStorage.setItem("activeCam",deviceId);
```

Comment:
Suggest making the key in local storage more identifiable for this node.
```suggestion
                            window.localStorage.setItem("node-red-node-ui-webcam-activeCam",deviceId);
```